Tensile/TensileLibLogicToYaml.py

```diff
@@ -242,19 +242,13 @@ def checkMacroTileThreadTileWorkGroupMatches(currentSolution):
    
     if WG1 !=  int(currentSolution["WorkGroup"][1]):
       raise RuntimeError("WorkGroup1 {0} does not match LibLogic value {1}".format(WG1, currentSolution["WorkGroup"][1]))
-
-    if matrixM !=  int(currentSolution["MatrixInstM"]):
-      raise RuntimeError("MatrixInstM {0} does not match LibLogic value {1}".format(matrixM, currentSolution["MatrixInstM"]))
-  
-    if matrixN !=  int(currentSolution["MatrixInstN"]):
-     raise RuntimeError("MatrixInstN {0} does not match LibLogic value {1}".format(matrixN, currentSolution["MatrixInstN"]))
      
 def calculateMatrixMNThreadTileMacroTileWorkGroupParameters(MIBlock,MIWaveTile,MIWaveGroup,waveFrontSize):

```

Comment:
No need to return matrixM,matrixN if we remove this part.
If we remove these return value, better to change the function name calculateMatrixMNThreadTileMacroTileWorkGroupParameters.